src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Scanner;
+
+import static java.lang.Integer.parseInt;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner myObj = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("what can i do for you");
+
+        String input = myObj.nextLine();
+        Todos[] todo = new Todos[100];
+        int c = 0;
+        while(!input.equalsIgnoreCase("bye")) {
+            if (input.equalsIgnoreCase("list")) {
+                System.out.println("-------TODO-LIST------");
+                System.out.println("----------------------");
+                for (int i = 0; i < todo.length && todo[i] != null; i++) {
```

Comment:
This seems like a very c-style implementation - you have a c and counter, you might want to consider looping until it reaches your counter instead. This will make your code cleaner.

Or, if you want to continue using this method, to make effective use of short-circuiting, you can switch your i < todo.length and todo[i] != null